catroidTest/src/org/catrobat/catroid/uitest/content/brick/SetLookBrickTest.java

```diff
@@ -207,7 +207,7 @@ public void testAddNewLook() {
 		solo.goBack();
 		solo.waitForFragmentByTag(LookFragment.TAG);
 
-		solo.sleep(3000);
+		solo.sleep(5000);
 		assertTrue("Testfile not added from mockActivity", solo.searchText(testFile));
```

Comment:
Use ```solo.waitForText(testFile);``` instead of ```solo.searchText(testFile)```. Then you can drop the solo.sleep(...) as well.
http://robotium.googlecode.com/svn/doc/com/robotium/solo/Solo.html#waitForText(java.lang.String)